packages/zoe/src/zoe.js

```diff
@@ -230,7 +275,7 @@ import { makeTables } from './state';
  * @param  {OfferHook} offerHook - a function that will be handed the
  * offerHandle at the right time, and returns a contract-specific
  * OfferOutcome which will be put in the OfferResultRecord.
- * @param  {CustomProperties} customProperties - an object of
+ * @param  {CustomProperties} [customProperties] - an object of
```

Comment:
Why the square brackets?

Reply:
It marks it as optional. (Displays the param with a `?`.)  I couldn't find a syntax using question mark directly.